client/me/purchases/remove-purchase/remove-domain-dialog/index.jsx

```diff
@@ -56,26 +55,6 @@ class RemoveDomainDialog extends Component {
 						}
 					) }
 				</p>
-				{ ! isRefundable( purchase ) && maybeWithinRefundPeriod( purchase ) && (
-					<p>
-						<strong>
-							{ translate(
-								"We're not able to refund this purchase automatically. " +
-									"If you're canceling within %(refundPeriodInDays)s days of " +
-									'purchase, {{contactLink}}contact us{{/contactLink}} to ' +
-									'request a refund.',
```

Comment:
Hm, I'm pretty sure we don't want to remove this yet.  Since we do still have major cases where users are owed a refund but the system can't process them, so until we handle that better I think we need to keep this message.

One case (maybe the only major one) is the one where you buy a plan and domain, cancel the plan, and then cancel the domain afterwards.  That case was mentioned in the review above (and just to follow up on the comment there, we do have that issue tracked internally; see my comments on D36311-code for more details).

Reply:
My above comment is no longer a concern, since all those issues are now being addressed server-side in D36311-code.

However, I realized while testing that because some of the server-side fixes change what happens when a plan is cancelled (and those changes are needed to make the domain refundable later), we'll have a roughly 96 hour window after those changes go live where some people will still have domains that should be refundable but can't be automatically refunded.  So it would be useful to delay releasing the changes in this file until around 96 hours after the server-side changes go live.  The rest of the pull request does not have to wait, though.

Reply:
Definitely worth waiting to publish that change.